src/Components/UserProfile/User.js

```diff
@@ -30,7 +30,13 @@ function User() {
     <>
       <Navbar
         start={
-          <Link to="/" aria-label="Go back to Home">
+          <Link
+            to="/"
+            aria-label="Go back to Home"
+            style={{
```

Comment:
This should not be needed in order to center the icon and title. I think there may be some random margin on the github icon/version number that shouldn't be there which is making these things flex wrongly.

Reply:
If the margin is not added to the github icon and the content is centered only using justify-content: space-between, the icon and title are not centered to the page properly.